inst/rmarkdown/templates/meningitis_outbreak/skeleton/skeleton.Rmd

```diff
@@ -606,24 +604,27 @@ linelist_cleaned <- linelist_cleaned %>%
 # the tilda (~) is used to assign the new values (Conf, prob, susp, unknown)
 # starting from the specific to the general
 # TRUE assigns all remaining rows 
-# linelist_cleaned <- linelist_cleaned %>%
-#   mutate(case_def = case_when(
-#     test_result == "Positive"                               ~ "Confirmed",
-#     test_result == "Negative" & maculopapular_rash == "Yes" ~ "Probable",
-#     test_result == "Negative"                               ~ "Suspected",
-#     TRUE                                                    ~ "Unknown"
-#   ))
+
+linelist_cleaned <- linelist_cleaned %>%
+  mutate(case_def = case_when(
+    grepl("Mening", pastorex_result)                        ~ "Confirmed",
+    fever == "Yes" &
```

Comment:
I think this could also be written this way to avoid the compound logic.

```r
fever == "Yes" & stiff_neck == "Yes" ~ "Probable",
fever == "Yes" & petechiae_purpura_at_admission == "Yes" ~ "Probable",
```

Reply:
Done!